.github/workflows/release.yaml

```diff
@@ -8,6 +8,7 @@ on:
 # https://github.com/ossf/scorecard/blob/7ed886f1bd917d19cb9d6ce6c10e80e81fa31c39/docs/checks.md#token-permissions
 permissions:
   contents: read
+  id-token: write
```

Comment:
add a comment here saying that this is for slsa attestations, and maybe a link to the docs?

Reply:
Done.